app/services/account_reset/delete_account.rb

```diff
@@ -20,7 +23,16 @@ def call
 
     private
 
-    attr_reader :success
+    attr_reader :success, :account_age, :mfa_methods
+
+    def track_account_age
+      @account_age = ((Time.zone.now - user.confirmed_at) / 1.day).round
+    end
+
+    def track_mfa_methods
+      @mfa_methods = MfaContext.new(user).two_factor_configurations.
```

Comment:
Another option here is to use the selection presenter: `TwoFactorLoginOptionsPresenter.new(user, nil, nil).options.map(&:method).uniq`

The `uniq` can be left off if you want the method to show up the same number of times as the user has configurations for that method.

This is probably using the presenter for a non-presentation purpose, but it has the benefit of capturing exactly the methods we would have presented to the user on login.